doc/kernel/memory_management/demand_paging.rst

```diff
@@ -118,12 +118,19 @@ Eviction Algorithm
 
 The eviction algorithm is used to determine which data page and its
 corresponding page frame can be paged out to free up a page frame
-for the next page in operation. There are two functions which are
+for the next page in operation. There are 4 functions which are
 called from the kernel paging code:
 
 * :c:func:`k_mem_paging_eviction_init()` is called to initialize
   the eviction algorithm. This is called at ``POST_KERNEL``.
 
+* :c:func:`k_mem_paging_eviction_add()` is called each time a data page becomes
+  eligible to future eviction.
+
+* :c:func:`k_mem_paging_eviction_remove()` is called when a data page is no
+  longer eligible to eviction. This may happen if given data page gets
```

Comment:
eligible to eviction -> eligible for eviction   (applies to line 128 as well).

if given -> if the given